src/applications/disability-benefits/all-claims/config/schema.js

```diff
@@ -797,6 +797,21 @@ const schema = {
             type: 'string',
             enum: ['NONE', 'NEW', 'SECONDARY', 'INCREASE', 'REOPEN'],
           },
+          specialIssues: {
+            type: 'array',
+            items: {
```

Comment:
Not likely to make a huge difference, but should probably have an `maxItems: 100` property to match swagger

Reply:
@markgreenburg : Change done to add maxItems to match the swagger